Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/WelcomeScreen.kt

```diff
@@ -120,7 +120,7 @@ private fun Branding(modifier: Modifier = Modifier) {
 @Composable
 private fun Logo(
     modifier: Modifier = Modifier,
-    lightTheme: Boolean = MaterialTheme.colors.isLight
+    lightTheme: Boolean = surfaceIsLight()
```

Comment:
Having this as the default value implies that this composable should basically only ever be displayed on top of a `Surface`, but there isn't anything in place to enforce that. I'd prefer no default value and this value passed through at the call site where a `Surface` is present. Same goes for other usages.

Reply:
Yeah I didn't love this, but actually your idea gave me another one that the two places where this is used really only care about what the content color should be (Surface just happens to be a place that could change that), so I updated the calling sites to specifically check the LocalContentColor.current instead. WDYT?

Reply:
I like the new approach! Perhaps we could use `LocalContentColor.current.luminance() < 0.5f` as the default value in this cases and keep the `lightTheme` parameter from before?

Reply:
Sounds good!